projects/bp-gallery/src/views/shared/picture-info/DescriptionsEditField.tsx

```diff
@@ -36,6 +37,36 @@ const DescriptionsEditField = ({
     [role]
   );
 
+  // This solution is necessary because of this jodit issue:
+  // https://github.com/jodit/jodit-react/issues/101
+  // problem: jodit keeps stale references to events
+  const onBlurCallback = useCallback(
+    (newText: string, description: FlatDescription) => {
+      onChange(
+        descriptionState
+          .map(d => (d === description ? { ...d, text: newText } : d))
+          .filter(description => !isEmpty(description.text))
+      );
+    },
+    [descriptionState, onChange]
+  );
+
+  const onChangeCallback = useCallback(
+    (newText: string, description: FlatDescription) => {
+      if (description.text !== newText) {
+        onTouch();
+      }
+    },
+    [onTouch]
+  );
+
+  const onBlurRef = useRef<(newText: string, description: FlatDescription) => void>(onBlurCallback);
+  const onChangeRef =
+    useRef<(newText: string, description: FlatDescription) => void>(onChangeCallback);
+
+  onBlurRef.current = onBlurCallback;
+  onChangeRef.current = onChangeCallback;
```

Comment:
Is that assignment to the current prop of the ref necessary if you pass the same callbacks as initial values to the `useRef` hook? 
Or is that the equivalent of updating the ref when the callback was newly created? And in this case: whats the difference to doing this in a `useEffect`? Hope, you can shine light on this 💡

Reply:
Alright, that makes it clear for me 👍